sdk/template/Azure.Template/tests/Samples/Sample1_HelloWorldAsync.cs

```diff
@@ -12,28 +12,8 @@
 
 namespace Azure.Template.Tests.Samples
 {
-    public class TemplateClientSamples: SamplesBase<TemplateClientTestEnvironment>
+    public partial class Sample1_HelloWorld: SamplesBase<TemplateClientTestEnvironment>
```

Comment:
Just call the class `Sample1` and avoid the underscores in names. Likely will trigger Roslyn errors. All the SampleN samples should be unique anyway - the words after are just to provide some meaning to the file names. The classes themselves aren't really paramount to the sample - just the method names.

Reply:
updated